ckanext/ogdchcommands/commands.py

```diff
@@ -266,6 +286,30 @@ def _get_datastore_table_page(self, context, offset=0):
 
         return (resource_id_list, has_next_page)
 
+    def cleanup_filestore(self, source=None):
+        """
+        command for cleaning up orphaned filestore-files if the
+        associated resource no longer exists.
+        """
+        user = logic.get_action('get_site_user')({'ignore_auth': True}, {})
+        context = {
+            'model': model,
+            'session': model.Session,
+            'user': user['name']
+        }
+        result = logic.get_action('ogdch_cleanup_filestore')(
+            context,
+            {
+                'dryrun': self.options.dryrun,
+            })
+        if self.options.dryrun:
+            print(msg_filestore_cleanup_dryrun
+                  .format(result.get('filepaths')))
+        else:
+            print(msg_filestore_cleanup
+                  .format(result.get('filepaths')))
```

Comment:
there should be another argument for `format()` here, probably `len(result.get.('filepaths))`